packages/venia-ui/lib/components/ForgotPassword/FormSubmissionSuccessful/formSubmissionSuccessful.js

```diff
@@ -3,16 +3,31 @@ import { shape, string } from 'prop-types';
 
 import { mergeClasses } from '../../../classify';
 import defaultClasses from './formSubmissionSuccessful.css';
+import { FormattedMessage, useIntl } from 'react-intl';
 
 const FormSubmissionSuccessful = props => {
     const { email } = props;
+    const { formatMessage } = useIntl();
     const classes = mergeClasses(defaultClasses, props.classes);
 
-    const textMessage = `If there is an account associated with ${email} you will receive an email with a link to change your password.`;
+    const textMessage = formatMessage(
+        {
+            id:
+                'If there is an account associated with {email} you will receive an email with a link to change your password.',
+            defaultMessage:
+                'If there is an account associated with {email} you will receive an email with a link to change your password.'
+        },
+        { email: email }
```

Comment:
You shouldn't need to pass `{ email: email }`, you should just be able to do `{ email }` as the keys match.